src/main/java/nl/novi/TechItEasy/controllers/TelevisionController.java

```diff
@@ -0,0 +1,116 @@
+package nl.novi.TechItEasy.controllers;
+
+import nl.novi.TechItEasy.Repositories.TelevisionRepository;
+import nl.novi.TechItEasy.exceptions.RecordNotFoundException;
+import nl.novi.TechItEasy.models.Television;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/televisions")
+public class TelevisionController {
+    @Autowired
+    TelevisionRepository televisionRepository;
```

Comment:
In de uitwerking is deze repository private en final, maar ik geloof niet dat we dat al in de les geleerd hadden.